specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2021-05-01-preview/mysql.json

```diff
@@ -0,0 +1,2604 @@
+          "type": "string",
+          "description": "indicates whether custom window is enabled or disabled"
+        },
+        "startHour": {
+          "type": "integer",
+          "format": "int32",
+          "description": "start hour for maintenance window"
+        },
+        "startMinute": {
+          "type": "integer",
+          "format": "int32",
+          "description": "start minute for maintenance window"
+        },
+        "dayOfWeek": {
+          "type": "integer",
+          "format": "int32",
+          "description": "day of week for maintenance window"
+        }
+      }
+    },
+    "Backup": {
+      "type": "object",
+      "properties": {
+        "backupRetentionDays": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Backup retention days for the server."
+        },
+        "geoRedundantBackup": {
+          "$ref": "#/definitions/EnableStatusEnum",
+          "description": "Whether or not geo redundant backup is enabled."
+        },
+        "earliestRestoreDate": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "Earliest restore point creation time (ISO8601 format)"
+        }
+      },
+      "description": "Storage Profile properties of a server"
+    },
+    "Storage": {
+      "type": "object",
+      "properties": {
+        "storageSizeGB": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Max storage size allowed for a server."
+        },
+        "iops": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Storage IOPS for a server."
+        },
+        "autoGrow": {
+          "description": "Enable Storage Auto Grow or not.",
+          "$ref": "#/definitions/EnableStatusEnum"
+        },
+        "storageSku": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The sku name of the server storage."
+        }
+      },
+      "description": "Storage Profile properties of a server"
+    },
+    "Sku": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the sku, e.g. Standard_D32s_v3."
+        },
+        "tier": {
+          "type": "string",
+          "description": "The tier of the particular SKU, e.g. GeneralPurpose.",
+          "enum": [
+            "Burstable",
+            "GeneralPurpose",
+            "MemoryOptimized"
+          ],
+          "x-ms-enum": {
+            "name": "SkuTier",
+            "modelAsString": true
+          }
+        }
+      },
+      "required": [
+        "name",
+        "tier"
+      ],
+      "description": "Billing information related properties of a server."
+    },
+    "Network": {
+      "type": "object",
+      "properties": {
+        "publicNetworkAccess": {
+          "$ref": "#/definitions/EnableStatusEnum",
+          "readOnly": true,
```

Comment:
why is this readonly?  The description implies it is settable and the contract for publicNetworkAccess typically has this settable

Reply:
In our design, the property of publicNetworkAccess is not settable, its value depends on if delegatedSubnetResourceId is provided or not when creating a server. If delegatedSubnetResourceId is provided in request body,  the value of publicNetworkAccess will be Disabled (server can only be accessed via vnet), else it is Enabled. 
	"network": {
            "delegatedSubnetResourceId": "xxxxx"
          }

Reply:
Please update the description of this property to reflect that

Reply:
updated in new commit